### PR TITLE
Application features

### PR DESCRIPTION
**Pull request by janegreene**
Completed user story 16 for application 
made new form that has flash message once submit is hit
removes pet from favorites after you apply for them

### CODE REVIEW

File: app/models/application.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Application < ApplicationRecord
+  # validates_presence_of :name
```
Comment (janegreene) [inline app/models/application.rb:2]: should delete comments

File: db/migrate/20200514194841_create_applications.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      t.string :city
+      t.string :state
+      t.integer :zip
+      t.string :phone
+      t.string :description
+
+      t.timestamps
+    end
+  end
+end
```
Comment (whitneykidd) [inline db/migrate/20200514194841_create_applications.rb:15]: Nice job on creating these tables 🎊

File: app/views/apps/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<div class="form_container">
+<h1>Create a New Application</h1>
+
+<%= form_tag "/applications/new" do %>
+
+<%= label_tag :pets, "Pets" %><br>
+
+<%= select_tag 'pet_ids[]', options_for_select(favorite.favorite_pets.map {|pet| [pet.name, pet.id]}), multiple: :true, required: :true %>
```
Comment (janegreene) [inline app/views/apps/new.html.erb:8]: will style this prior to turn in

File: spec/features/applications/new_spec.rb [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    fill_in "Phone", with: "918-222-2321"
+    fill_in "Description", with: "Very few alligators in my yard..."
+    click_button "Submit Application"
+    expect(current_path).to eq("/favorites")
+    expect(page).to have_content("You have submitted your application to adopt.") #needs rework
+    expect(page).to_not have_content("#{pet1.name}")
+    expect(page).to_not have_content("#{pet2.name}")
+  end
+end
+# User Story 16, Applying for a Pet
```
Comment (janegreene) [inline spec/features/applications/new_spec.rb:52]: will delete prior to final submission

File: app/controllers/apps_controller.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class AppsController < ApplicationController
+
+  def new
+    @favorites = favorite.favorite_pets
+  end
+
+  def create
+    @apps = Application.create
+      params[:pet_ids].each do |id|
```
Comment (whitneykidd) [inline app/controllers/apps_controller.rb:9]: Will have to add if/else statement for upcoming stories here

**Pull request merged**